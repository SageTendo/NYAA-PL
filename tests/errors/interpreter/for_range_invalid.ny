uWu_nyaa() => {
    for _ => (3.3, "3") {
        pasu
    }
}