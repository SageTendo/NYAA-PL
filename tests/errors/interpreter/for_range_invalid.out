cannot be used as an integer